### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cs-adarsh-gupta**
#### UTCs: 
 
- [x] Connector installation verified. 
- [x] Connector logo verified. 
- [x] Actions and Playbook list verified. 
- [x] Playbook tags verified. 
- [x] All Playbook are in info mode verified. 
- [x] All Playbook are in inactive mode verified.

### CODE REVIEW

File: akamai-waf/connector.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Copyright start
+MIT License
+Copyright (c) 2024 Fortinet Inc
+Copyright end
+"""
+
+from connectors.core.connector import Connector, get_logger, ConnectorError
+from .operations import operations, _check_health
+
+logger = get_logger('akamai-waf_1_0_0')
```
Comment (cs-omkar-kumbhar) [inline akamai-waf/connector.py]: you can just write logger = get_logger('akamai-waf') here, this change can be done in all files

File: akamai-waf/operations.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        logger.error(str(err))
+        raise ConnectorError(str(err))
+
+
+def _check_health(config):
+    try:
+        aw = AkamaiWAF(config)
+        endpoint = "/network-list/v2/network-lists"
+        response = aw.make_request(endpoint=endpoint)
+        if response.ok:
```
Comment (cs-omkar-kumbhar) [inline akamai-waf/operations.py:190]: in else you can return false

File: akamai-waf/requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+edgegrid-python
```
Comment (cs-omkar-kumbhar) [inline akamai-waf/requirements.txt]: you can specify edgegrid-python>='library version'

File: akamai-waf/info.json [hunk 4]
```diff
@@ -0,0 +1,848 @@
+{
+  "name": "akamai-waf",
+  "label": "Akamai WAF",
+  "version": "1.0.0",
+  "description": "Akamai Web Application Firewall (WAF) is a cloud-based security solution designed to protect web applications from various cyber threats. It acts as a shield between internet traffic and web servers, inspecting incoming requests to detect and block malicious activities",
+  "publisher": "Fortinet",
+  "cs_approved": true,
```
Comment (cs-shubham-morankar) [inline akamai-waf/info.json]: If connector is uncertified, then pls update "cs_approved" value as false

**Review by cs-shubham-morankar — APPROVED** [review]
LGTM

**Review by cs-omkar-kumbhar — APPROVED** [review]

**Pull request merged**